src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -2977,11 +2999,15 @@ private String getMailArchiveMessageFolderName(JSONObject message, String channe
 			{
 				messageFolderName = channelName + Utils.PATH_SEPARATOR;
 			}
-			String name = messageFolderName + " " + date + " " + sender + " " + subject;
-			if(name.length()>100){
-				name=name.substring(0,100);
+			// If there is a channel then put in the separator space.
+			if (! messageFolderName.equals("")) {
+				messageFolderName += " ";
 			}
-			messageFolderName = Utils.sanitizeName(Utils.COLLECTION_TYPE, name) + "/";
+			messageFolderName += date + " " + sender + " " + subject;
+			if(messageFolderName.length()>100){
+				messageFolderName=messageFolderName.substring(0,100);
```

Comment:
No, but that section is existing code with a new variable name.  Changing that would be a new jira.